scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -3114,38 +3105,39 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) {
         name match {
           case q: Quasi if endTparamQuasi => q.become[Type.Param]
           case _ =>
-            val tparams = typeParamClauseOpt(ownerIsType = true, ctxBoundsAllowed = false)
+            val tparams = typeParamClauseOpt(ownerIsType = true)
             val tbounds = typeBounds()
             val vbounds = new ListBuffer[Type]
             val cbounds = new ListBuffer[Type]
-            if (ctxBoundsAllowed) {
-              @inline
-              def getBound(): Type = {
-                next()
-                currToken match {
-                  case t: Ellipsis => ellipsis[Type](t, 1)
-                  case _ => typ()
-                }
-              }
-              if (at[Viewbound]) {
-                if (!dialect.allowViewBounds) {
-                  val msg = "Use an implicit parameter instead.\n" +
-                    "Example: Instead of `def f[A <% Int](a: A)` " +
-                    "use `def f[A](a: A)(implicit ev: A => Int)`."
-                  syntaxError(s"View bounds are not supported. $msg", at = currToken)
-                }
-                doWhile(vbounds += getBound())(at[Viewbound])
-              }
-              while (at[Colon]) cbounds += getBound()
+            @inline
+            def getBound(allowAlias: Boolean): Type = currToken match {
+              case t: Ellipsis => ellipsis[Type](t, 1)
+              case _ => contextBoundOrAlias(allowAlias)
+            }
+            if (acceptOpt[Viewbound])
+              doWhile(vbounds += getBound(allowAlias = false))(acceptOpt[Viewbound])
```

Comment:
hmm, perhaps this might be simpler, sorry i didn't realize it earlier:
```
while (acceptOpt[Viewbound])
  vbounds += getBound(allowAlias = false)
```

Reply:
Fixed!